docs/docs/build/getting-started/helloworld.mdx

```diff
@@ -44,91 +51,419 @@ Follow the steps from the [foundry book](https://book.getfoundry.sh/getting-star
+          bytes32 scriptPubKeyHash =
+          keccak256(abi.encodePacked(uint8(bitcoinAddress.scriptPubKey.length), bitcoinAddress.scriptPubKey));
+
+          uint256 txOutputValue = BitcoinTx.getTxOutputValue(scriptPubKeyHash, transaction.outputVector);
+
+          require(txOutputValue >= expectedBtcAmount, "Bitcoin transaction amount is lower than in accepted order.");
+      }
+  }
 
+```
   </TabItem>
   <TabItem value="foundry" label="Foundry">
 
-Start a new project with Foundry.
+Clone the starters toolkit provided
 
 ```shell
-forge init coin
+git clone https://github.com/bob-collective/DevStarterKit.git
+cd DevStarterKit
 ```
 
-Create a new `Coin.sol` file in src directory.
-
+The `HelloBitcoin` contract should already be present inside the src directory. To create a new contract:
 ```shell
-cd coin
-touch Coin.sol
+touch src/<Contract_Name>.sol
 ```
 
   </TabItem>
 </Tabs>
 
-Enter the below code in `Coin.sol` file. To learn more about the contract [checkout the Solidity tutorial guide](https://docs.soliditylang.org/en/v0.8.21/introduction-to-smart-contracts.html#subcurrency-example).
-
-```solidity
 
-// SPDX-License-Identifier: GPL-3.0
-pragma solidity ^0.8.4;
-
-contract Coin {
-    // The keyword "public" makes variables
-    // accessible from other contracts
-    address public minter;
-    mapping(address => uint) public balances;
-
-    // Events allow clients to react to specific
-    // contract changes you declare
-    event Sent(address from, address to, uint amount);
-
-    // Constructor code is only run when the contract
-    // is created
-    constructor() {
-        minter = msg.sender;
-    }
-
-    // Sends an amount of newly created coins to an address
-    // Can only be called by the contract creator
-    function mint(address receiver, uint amount) public {
-        require(msg.sender == minter);
-        balances[receiver] += amount;
-    }
-
-    // Errors allow you to provide information about
-    // why an operation failed. They are returned
-    // to the caller of the function.
-    error InsufficientBalance(uint requested, uint available);
-
-    // Sends an amount of existing coins
-    // from any caller to an address
-    function send(address receiver, uint amount) public {
-        if (amount > balances[msg.sender])
-            revert InsufficientBalance({
-                requested: amount,
-                available: balances[msg.sender]
-            });
-
-        balances[msg.sender] -= amount;
-        balances[receiver] += amount;
-        emit Sent(msg.sender, receiver, amount);
-    }
-}
-```
-
-### Compile the Coin Contract
+### Compile the Contract
 
 <Tabs groupId="dev-env">
   <TabItem value="remix" label="Remix">
 
-To compile contract go to `Solidity Compiler` section of the IDE, select and compile the `Coin` smart contract.
+In Advance Configration switch from `Compiler configuration` to `Use configuration file`.
```

Comment:
Is this Advance Configuration or Advanced Configuration?

Reply:
`Advanced Configurations` updated it.